src/cascadia/TerminalApp/Pane.cpp

```diff
@@ -1237,6 +1225,14 @@ void Pane::UpdateVisuals()
 // - <none>
 void Pane::_Focus()
 {
+    // Don't focus our content if we're already focused. This prevents a bug
+    // where tapping on the arrow in a ComboBox will land in our Tapped handler,
+    // and if we steal focus from the ComboBox, it won't open. See GH#17062
+    if (WasLastFocused())

```

Comment:
Is there a way for us to check if focus **is currently inside** our pane? Rather than us trying to remember it.